README.md

```diff
@@ -15,14 +15,29 @@
 -->
 # Examples
 
-The example demonstrates different users querying an avro file over a REST api.
+The example demonstrates different users querying an avro file over a REST api. These examples are run in 3 different scenarios as described in the [Deployment](deployment) and [Deployment-jvm](deployment-jvm) modules:
```

Comment:
Prefer line-breaks in markdown every sentence
```suggestion
The example demonstrates different users querying an avro file over a REST api.
These examples are run in 3 different scenarios as described in the [Deployment](deployment) and [Deployment-jvm](deployment-jvm) modules:
```

Reply:
Updated